### ISSUE TITLE
Framebuffer is incomplete: Attachments are not all the same size.

### ISSUE THREAD

**Issue by Friksel** [jnsmalm/pixi3d#97]
Using `PostProcessingSprite()` sometimes the sprite just doesn't get rendered. It's than throwing these kinde of errors:
![image](https://user-images.githubusercontent.com/3089978/170704209-d8e5eac4-4496-401b-b55c-77b93c1f8988.png)

I'm not using any textures on the 3d model, so it seems like it's talking about the PostProcessingSprite itself as being a texture.

I've seen other people with the same error message on the pixi github, but to me it's completely unclear what is causing this and how this could even be worked around. I'm trying to debug this at the moment.

Any clue on what could cause this issue to happen? Could it be related to using the PostProcessingSprite's `resolution`, `width` or `height` settings?

**Comment by Friksel** [comment]
Could it perhaps be related to this issue from years ago?:

https://github.com/pixijs/pixijs/issues/746

**Comment by jnsmalm** [comment]
Do you have an example which sometimes reproduces this error?

**Comment by Friksel** [comment]
@jnsmalm Sorry, can't send you the project. When debugging I would try to create a new, small, project to isolate an issue. But the thing is that yesterday when I tried to debug this the issue didn't happen once and couldn't replicate it. While at other times it happens every two or third time the model is shown. It seems completely random.

I did change a little thing now tho, I was first changing the resolution of the sprite to a value that had a lot of decimals, because Javascript didn't seem to be able to do a simple calculation like `41*0.01`. It didn't result in `0.41`, but in `0.41000000000000003` instead. Which to me is weird (try it yourself in a Chrome dev console: `console.log(41*0.01)`. Now I'm using toFixed(2) to round instead. As you understand, it would be weird if this would be the reason of the issue, but who knows.

Long story short, it's a little difficult to create a small demo if not sure what exactly to isolate and when exactly this issue appears so we can replicate it.
